weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallsPage/CallsTableButtons.tsx

```diff
@@ -314,6 +359,41 @@ const DownloadGrid: FC<{
           Export to JSON
         </ClickableOutlinedCardWithIcon>
       </div>
+      <div className="mt-8 flex items-center">
+        <ClickableOutlinedCardWithIcon
+          iconName="python-logo"
+          onClick={() => {
+            setCodeMode('python');
+            navigator.clipboard.writeText(pythonText);
+          }}>
+          <span className="w-full">Use Python</span>
+          <div className="flex w-full justify-end">
+            <Icon name="copy" size="small" />
+          </div>
+        </ClickableOutlinedCardWithIcon>
+        <div className="ml-8" />
+        <ClickableOutlinedCardWithIcon
+          iconName="code-alt"
+          onClick={() => {
+            setCodeMode('curl');
+            navigator.clipboard.writeText(curlText);
```

Comment:
I'm having mixed feelings about autocopy to clipboard, did this come from design?

Reply:
Okay, I see your point. Lets leave this out and then circle back with design. @jamie-rasmussen